_includes/configuration/fullscreen-native.md

```diff
@@ -0,0 +1,35 @@
+## `fullscreen_native`
```

Comment:
These heading levels appear to be wrong? `Configuration options` is h2 and this should be rendered as h2 as well.

Reply:
Checked launcher and can confirm this an all other existing settings are wrong (only because you added the subheading for the configuration, which makes sense).

Reply:
Oops I looked at `Full page`, not `Fullscreen` sorry :facepalm: Either way, this still seems like a bug?

Reply:
nope `<h3>` is correct see: https://github.com/tinymce/tinymce-docs/pull/1709/files#diff-13e7545cec62f3f5f260a87fc02535064f43b07dbcbf38db93dabfbfb96319d1R37 there's a `#` in front of the include to shift it down one level (unfortunately only affects the first heading)

Reply:
As discussed, this isn't very good, but it's like that elsewhere so out of scope of this JIRA.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1709/commits/d7983cc3d6e57d42769ad40890f39472cbd61bab